apis/nucleus/src/components/Cell.jsx

```diff
@@ -351,6 +351,7 @@ const Cell = forwardRef(
       translator,
       language,
       keyboardNavigation,
+      externalFocusManagement,
```

Comment:
consider making it possible to set a generic / default? blurCallback(?) for the externalFocusManagement case

Reply:
yea, this might evolve to that, we'll see